checker.go

```diff
@@ -36,62 +45,40 @@ type FullScore struct {
 
 var lettersAndSpaceOnly = regexp.MustCompile(`[^a-zA-Z ]+`)
 
-// Negativity calculates the negative sentiment of a sentence
-func Negativity(phrase string) Score {
+func calculateScore(phrase string, calcSign sign) Score {
 	var hits float64
 	var words []string
+	var count int
 
-	addPush := func(t string, score float64) {
-		hits -= score
-		words = append(words, t)
-	}
-
+	var lettersAndSpaceOnly = regexp.MustCompile(`[^a-zA-Z ]+`)
```

Comment:
Why not keep the global one? It is concurrent-safe, and it avoids Regex compilation. No?